### PR TITLE
blockchaincom.fetchMarkets unified

### PR DESCRIPTION
**Pull request by samgermain**
```
2022-01-18T03:39:11.718Z
Node.js: v14.17.0
CCXT v1.68.25
blockchaincom.fetchMarkets ()
10249 ms
        id | numericId |     symbol |  base | quote | baseId | quoteId | type | spot | margin |  swap | future | option | active | contract | linear | inverse | contractSize | expiry | expiryDatetime | strike | optionType |                          precision |                                                          limits
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   XLM-EUR |        11 |    XLM/EUR |   XLM |   EUR |    XLM |     EUR | spot | true |  false | false |  false |  false |   true |    false |        |         |              |        |                |        |            |    {"price":0.00001,"amount":1e-7} |        {"leverage":{},"amount":{"min":70},"price":{},"cost":{}}
...
   DAI-USD |        74 |    DAI/USD |   DAI |   USD |    DAI |     USD | spot | true |  false | false |  false |  false |   true |    false |        |         |              |        |                |        |            |   {"price":0.0001,"amount":0.0001} |         {"leverage":{},"amount":{"min":5},"price":{},"cost":{}}
 NEAR-USDT |       110 |  NEAR/USDT |  NEAR |  USDT |   NEAR |    USDT | spot | true |  false | false |  false |  false |  false |    false |        |         |              |        |                |        |            |       {"price":0.01,"amount":1e-8} |       {"leverage":{},"amount":{"min":0.1},"price":{},"cost":{}}
107 objects
```

### CODE REVIEW

File: js/blockchaincom.js [hunk 1]
```diff
@@ -229,33 +243,54 @@ module.exports = class blockchaincom extends Exchange {
...
+                    'price': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'cost': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                },
+
```
Comment (kroitor) [inline js/blockchaincom.js:293]: We cannot have empty lines inside methods, let's always build and test: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-build--run-tests-on-your-local-machine

Comment (samgermain) [reply js/blockchaincom.js]: Is there any reason for me NOT to just push it and let Travis and continuous-integration/appveyor/pr do the testing?

It takes about 5-10 minutes to test, and I can't switch branches while that's happening, that adds up, I pushed almost 10 branches yesterday

Comment (kroitor) [reply js/blockchaincom.js]: I mean a simple syntax check prior to pushing would save your time in this case, but it's up to you. Sure we can use Travis for that.

Comment (samgermain) [reply js/blockchaincom.js]: I have a live syntax checker, but it's not working for blank lines for some reason

Comment (samgermain) [reply js/blockchaincom.js]: How can I view the build? I want to check what rule was broken

**Pull request merged**